specification/azurestackhci/resource-manager/readme.md

```diff
@@ -64,14 +64,30 @@ suppressions:
     reason: The linter is mistakenly thinking that paths for a singular resource that is always named default, like "/{resourceUri}/providers/Microsoft.AzureStackHCI/virtualMachineInstances/default," is for a collection of resources.
```

Comment:
Looking at this old suppression, I don't think its supposed to be needed. Can we remove it?

> code: GetCollectionOnlyHasValueAndNextLink
    reason: The linter is mistakenly thinking that paths for a singular resource that is always named default, like "/{resourceUri}/providers/Microsoft.AzureStackHCI/virtualMachineInstances/default," is for a collection of resources.

Reply:
I wonder if the leading slash before '{resourceUri}': i.e. '/{resourceUri}' i the swagger paths, might be the actual cause of some of these linter errors.

Does removing that make any difference?

Reply:
@wyatt-troia Does it make any difference without the leading slash?

Reply:
I don't know. I never tried that

Reply:
I gave it a try locally, and the linter actually doesn't allow for paths without a leading slash.

Reply:
suppression removed. Fixed.

Reply:
Unresolving pending below discussion